File: app/src/main/java/workshop/akbolatss/xchangesrates/screens/charts/ChartView.java
package workshop.akbolatss.xchangesrates.screens.charts;

import workshop.akbolatss.xchangesrates.base.BaseView;
import workshop.akbolatss.xchangesrates.base.LoadingView;
import workshop.akbolatss.xchangesrates.model.response.ChartResponseData;

/**
 * Author: Akbolat Sadvakassov
 * Date: 04.01.2018
 */

public interface ChartView extends BaseView, LoadingView {

    void onLoadLineChart(ChartResponseData chartData);
}

File: app/src/main/java/workshop/akbolatss/xchangesrates/screens/charts/ChartView.kt
package workshop.akbolatss.xchangesrates.screens.charts

import workshop.akbolatss.xchangesrates.base.BaseView
import workshop.akbolatss.xchangesrates.model.response.ChartData

/**
 * Author: Akbolat Sadvakassov
 * Date: 04.01.2018
 */

interface ChartView : BaseView {

    fun onLoadLineChart(chartData: ChartData)
    fun toast(message: String)
    fun onSaveSnapshot(isSuccess: Boolean, chartData: ChartData)
}
